src/plugins/workspace/public/components/workspace_initial/workspace_use_case_card.scss

```diff
@@ -0,0 +1,80 @@
+/* stylelint-disable */
+.observability {
```

Comment:
The className may match unexpected elements, can we add some prefix?

Reply:
Sure, I will update it.